<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comteledialg</class>
 <widget class="QWidget" name="Comteledialg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>91</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="portNameComboBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="baudRateComboBox">
      <property name="currentIndex">
       <number>12</number>
      </property>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>75</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>110</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>134</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>150</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>56000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>76800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="dataBitsComboBox">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="parityComboBox">
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奇</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>偶</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="stopBitsComboBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="actionOpenCom">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionCloseCom">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendmsgBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送数据</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendMsgLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearUpBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清屏</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="stopBtn">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="backBtn">
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="up_leftBtn">
      <property name="text">
       <string>up-left</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="back_leftBtn">
      <property name="text">
       <string>back-left</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="upBtn">
      <property name="text">
       <string>up</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="leftBtn">
      <property name="text">
       <string>left</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="rightBtn">
      <property name="text">
       <string>right</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="back_rightBtn">
      <property name="text">
       <string>back-right</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="up_rightBtn">
      <property name="text">
       <string>up-right</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
